tests/evaluation/metrics_test.py

```diff
@@ -0,0 +1,134 @@
+"""Tests for GtsfmMetric and GtsfmMetricsGroup. 
+
+Authors: Akshay Krishnan
+"""
+import copy
+import numpy as np
+import unittest
+import tempfile
+import os
```

Comment:
Needs to be listed in three groups. I think not sorted alphabetically currently.

Reply:
updated, please let me know if the order is incorrect.

Reply:
thanks. I think it should go
```python
import copy
import os
import tempfile
import unittest
```
right?

Reply:
Yeah, my bad. Not sure why sublack did not format it that way though.

Reply:
no problem